oidc-controller/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10
```

Comment:
Could we maintain the same pattern as before, and have a `docker` folder in the root with all the Docker resources (Dockerfiles, docker-compose, configurations mounted as volumes, etc.) and the helper `manage` script? This would be consistent with the old pattern used in the repo (making the transition simpler), as well as the pattern used for other DTS apps.

Reply:
Got a `manage` file, and no .env file. Please take another look and let me know. Would be happy to change that in a follow up branch.